src/plotting/plot_mysql_sensor_sat_amv.py

```diff
@@ -69,13 +69,13 @@ def get_source_dir(row):
 print('connecting to mysql db')
 mysql_conn = itf.engine.connect()
 #BUFR FILE INFO
-sql = f"""select m.*, o.parent_dir from obs_meta_nceplibs_bufr as m inner join obs_inventory as o on m.obs_id = o.obs_id where o.s3_bucket = \'noaa-reanalyses-pds\'"""
+sql = f"""select m.*, o.parent_dir from obs_meta_nceplibs_bufr as m inner join obs_inventory as o on m.obs_id = o.obs_id where o.s3_bucket = \'noaa-reanalyses-pds\' AND o.parent_dir LIKE '%/amv/%' """
```

Comment:
For these like statements, can you add the full parent file path where possible? That way only one wildcard has to be evaluated

Reply:
Changes are all made and tested